test/autoCompletion.test.ts

```diff
@@ -386,6 +386,62 @@ describe('Auto Completion Tests', () => {
           .then(done, done);
       });
 
+      it('Autocomplete without default value - not required', async () => {
+        const languageSettingsSetup = new ServiceSetup().withCompletion();
+        languageSettingsSetup.languageSettings.disableDefaultProperties = true;
+        languageService.configure(languageSettingsSetup.languageSettings);
+        languageService.addSchema(SCHEMA_ID, {
+          type: 'object',
+          properties: {
+            scripts: {
+              type: 'object',
+              properties: {
```

Comment:
Can we possibly have a second non-string property added to this test case?

Reply:
I'm sorry for the longer delay...
I modified the UT. Have a look if I understood correctly your note, please.
vscode-yaml PR is here https://github.com/redhat-developer/vscode-yaml/pull/667